core/src/main/java/com/onelogin/saml2/http/HttpRequestUtils.java

```diff
@@ -0,0 +1,83 @@
+package com.onelogin.saml2.http;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class HttpRequestUtils {
```

Comment:
It's used in `toolkit` but not `core`.

However, I kept the utils in `core` so that they stay near the `HttpRequest` and `HttpResponse` interfaces given the close relationship between these classes and their usage:

![image](https://github.com/SAML-Toolkits/java-saml/assets/1202420/25a67bb7-687a-4fa0-bbde-639c67f7e0da)